rules/app.bzl

```diff
@@ -56,7 +56,7 @@ def ios_application(name, apple_library = apple_library, **kwargs):
     """
     infoplists = write_info_plists_if_needed(name = name, plists = kwargs.pop("infoplists", []))
     application_kwargs = {arg: kwargs.pop(arg) for arg in _IOS_APPLICATION_KWARGS if arg in kwargs}
-    library = apple_library(name = name, namespace_is_module_name = False, platforms = {"ios": application_kwargs.get("minimum_os_version")}, **kwargs)
+    library = apple_library(name = name, namespace_is_module_name = False, platforms = {"ios": application_kwargs.get("minimum_os_version")}, add_swift_local_debug_options = True, **kwargs)
```

Comment:
should also be passed for tests as well?

Reply:
Alternatively, we could only add in the select to the `deps` passed to `rules_apple_ios_application `, and make it conditional on whether there were actually any swift files? That way, the swiftmodule that represents the app needn't be invalidated by switching the flag either

Reply:
as of Xcode 11.5, we don't need this work around for apps, only tests.